src/mongo/db/pipeline/document_source_group.h

```diff
@@ -147,15 +147,21 @@ class DocumentSourceGroup final : public DocumentSource, public SplittableDocume
      */
     Value expandId(const Value& val);
 
+    struct Field {
+        std::string _fieldName;
```

Comment:
Since this is a simple struct, and the member variables are public, the names should not start with an _.